packages/web-components/fast-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+  accentFillRestBehavior,
+  heightNumber,	
+  neutralForegroundRestBehavior
+} from "../styles/index";
+
+export const AvatarStyles = css`
+  ${display("flex")} 
+  :host {
+    max-width: calc(${heightNumber} * 2.5px);
+    height: calc(${heightNumber} * 2.5px);
```

Comment:
Curious if we might want to add a a variable here that would allow us to control the size if we wanted. We could default the variable to this value?